### ISSUE TITLE
[Bug]: sp-number-field pressing increment/decrement buttons with VO on iOS double presses buttons

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3274]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-number-field

### Expected behavior

should be able to press buttons to increase/decrease numbers in sp-number-field

### Actual behavior

pressing the buttons seems to refresh the page and moves focus AND double presses the buttons

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/number-field/
2. start VO on iOS
3. tap on the number field
4. swipe right to move to the increase button
5. double tap to press the button
6. note the keyboard activates then deactivates
7. Focus moves to the "Show sidebar" button at the top of the page
8. The value increases by 2 steps

**Comment by najikahalsema** [comment]
#3272 #3271  

These bugs are all related to the inaccessibility of the increment/decrement buttons. These will need to be tested even more thoroughly than they have before. We test for all kinds of interactions, but not the kinds that include voiceover.

**Comment by TarunAdobe** [comment]
Hey @jnurthen I am trying to test this bug on my phone and it seems like everything is working fine... Can you verify from your end and if there's still an issue... can you please attach a screen-recording like mine so that I can verify and start working on it.

https://github.com/adobe/spectrum-web-components/assets/137032355/7715399d-8903-451f-a772-64500d0c0d6c

**Comment by jnurthen** [comment]
This seems to no longer reproduce

**Comment by jnurthen** [comment]
Actually - just tested again. It seems to reproduce on my iPad but not my iPhone. I'll record a testcase and do some more investigation.

**Comment by jnurthen** [comment]
https://github.com/adobe/spectrum-web-components/assets/1724479/e28e51c9-f4b6-4308-807d-f9bd99769c52

**Comment by jnurthen** [comment]
Additional info
Does not reproduce on iOS 16.6 w/ Bluetooth keyboard
Does reproduce on iPadOS 17.0 beta w/ Bluetooth keyboard

**Comment by jnjosh** [comment]
@TarunAdobe I tested this out on my iPad mini using iPad Os 16.6.1 with and without a Bluetooth keyboard. The interesting thing is I see this reproduce with and without the keyboard—it's just worse I think with the keyboard.

In the first video, with a Bluetooth keyboard connected (33 seconds) the button seems to actually trigger many times—going all the way up to +7. Interestingly it refocuses on the address bar.

https://github.com/adobe/spectrum-web-components/assets/66142/0f74db5a-262c-4e0d-aeda-14cdc5d0c31c


In the second video, without a Bluetooth keyboard connected (15 seconds) the button does trigger twice (going from -1 to +1). Then you can see the software keyboard flash, then refocus on the address bar again.


https://github.com/adobe/spectrum-web-components/assets/66142/9e673e8b-3601-4812-a398-adc1557809d7

**Comment by najikahalsema** [comment]
SWC-26

**Comment by caseyisonit** [comment]
Hello @jnurthen in order to help us triage your issue, [can you please review our bug severity classification guide](https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md#issue-severity-classification) and provide us with the appropriate severity classification? Please update the label to the correct severity level and remove `SEV missing` label.

If we don't hear from you in 2 weeks, we will close this issue as not planned.

<img width="1024" height="1028" alt="Image" src="https://github.com/user-attachments/assets/d5fd78ee-6e8e-4967-97f3-4e4198b79f13" />
